src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/LoggerMessageGenerator.Emitter.cs

```diff
@@ -612,7 +612,7 @@ private static string ConvertEndOfLineAndQuotationCharactersToEscapeForm(string
         /// Checks if variableOrTemplateName contains a special symbol ('@') as starting char
         /// </summary>
         /// <param name="variableOrTemplateName">variable that might contain '@' symbol</param>
-        /// <returns>true if contains special symbol, false otherwise.</returns>
+        /// <returns><see langword="true" /> if contains special symbol, false otherwise.</returns>
```

Comment:
This is a private method, not too important. 

```suggestion
        /// <returns><see langword="true" /> if it starts with the <c>@</c> symbol, <see langword="false"/> otherwise.</returns>
```